### DISCUSSION TITLE
Selected solution not working fully when tried out in plain html file

### DISCUSSION THREAD

**Discussion by sazzad88** [cssbattle/forum#385] [General]
Hi,
I have tried to see the solution in a better view in my code editor. But in css battle solution tab the code seems to be broken.
It is not working when i pasted that code in plain html.

<img width="1340" alt="image" src="https://github.com/cssbattle/forum/assets/33688148/9289480a-eeaa-4e7e-b0bd-4683bc7c6bf2">

What am i missing?
Any help is appreciated.

**Comment by SelenIT** [comment]
Most likely it's because of different viewport (browser window) size. All CSSBattle targets have the same fixed size of 400x300 pixels, and many solutions require this exact size to work (responsiveness is often the first thing to sacrifice for making code shorter). E.g., in this case, `padding:82 42` on the root element (targeted with the `&` selector) combined with default `8px` margin of `body` make the `body` element 400 - (42 + 8)*2 = 300px wide and 300 - (82 + 8)*2 = 120px tall, but these dimensions would be different with different window size.
